base/setup/lib/utils/partlist.c

```diff
@@ -564,8 +546,8 @@ IsSuperFloppy(
 
 
 /*
- * Inserts the disk region represented by PartEntry into either the primary
- * or the logical partition list of the given disk.
+ * Inserts the disk region represented by PartEntry into either
+ * the primary or the logical partition list of the given disk.
```

Comment:
I know you don't like to hear it, but I will mention it anyway: this is a superfluous change, that makes the PR longer than necessary.